lib/client.js

```diff
@@ -174,6 +177,39 @@ var GiantSwarm = (function () {
     },
 
     /**
+     * Internal function to perform an authenticated GET request and return the raw body
+     *
+     * @param {String} [uri] API route to be called, starting with "/"
+     * @param {Function} [successCallback] Callback to be called in case of success, gets response data as argument
+     * @param {Function} [errorCallback] Optional callback to be called in case of an error, error object as argument
+     * @param {Object} [params] URL parameters to be added (optional)
+     */
+    _simple_text_get_request: function(uri, successCallback, errorCallback, params) {
+      this._validateStringParameters({uri: uri});
+      if (!successCallback) throw new this.UserException("Parameter successCallback must be given");
+      if (typeof(successCallback) !== 'function') throw new this.UserException("Parameter successCallback must be of type function");
+      if (errorCallback !== null && typeof(errorCallback) !== 'function') throw new this.UserException('errorCallback must be null or function');
+      var headers = {
+        'Authorization': 'giantswarm ' + authToken
+      }
+      if (clusterId) {
+        headers[CLUSTER_ID_HEADER] = clusterId;
```

Comment:
Why is clusterId a global variable and not read from `this`? Would make it nicer.